corehq/apps/app_manager/templates/app_manager/partials/bulk_upload_app_translations.html

```diff
@@ -17,5 +17,15 @@
         </select>
       </p>
     </li>
+    {% if request|toggle_enabled:'APP_TRANSLATIONS_WITH_TRANSIFEX' %}
+    <li>
+      <p>
+        {% trans "Skip blacklisted translations?" %}
```

Comment:
My understanding is that this should exclude
1. blacklisted translations
2. translations that have "Skipped transifex" set on them, which includes labels, forms or complete modules. Check out `AppTranslationsGenerator`

Currently this PR is addressing just the first one but needs to skip both

If that is true, this sentence can be updated to say
"Exclude translations not sent for transifex" or
"Download translations eligible for transifex only"

Blacklisted translation was the name used for same purpose, so i think just saying that should be fine.